### ISSUE TITLE
Real time DOGE price stats page

### ISSUE THREAD

**Issue by akhileshthite** [dogecoin/dogecoin.com#239]
We can make a new html page showing real time prices of the dogecoin. That would be great for users to track the live dogecoin prices on this official dogecoin website.
If you know or want to suggest any official API for this then it would be great.
I would like to work on this one, I'll do the job with proper indentation and responsibility.

Thank you!

**Comment by aarsxx** [comment]
how about use tradingview.com to display the chart ?

**Comment by cluzier** [comment]
Is someone currently working on this? If not I will gladly take this on, just let me know.

**Comment by Aivolt1** [comment]
I'm happy to help with this project

**Comment by Aivolt1** [comment]
Can any one help me build a html site that identifies the source of any tx block or wallet address?? Using doge chain?

**Comment by PasserDreamer** [comment]
No, the Core Values of the dogecoin is "1 dogecoin = 1 dogecoin".
Take these things away; do not make doge Father's house a house of trade.

**Comment by Aivolt1** [comment]
I just want help building a site that will allow user to identify any block chain transactions on one platform ...? If users are happy with the service they can tip in any crypto however, the main goal is to put all the funds accumulated from the terminal will only be used to support DOGE community.

**Comment by Dogeyyboi** [comment]
I'll help!
It sounds awesome!

**Comment by akhileshthite** [comment]
> I'll help!
> It sounds awesome!

Thanks :)

**Comment by akhileshthite** [comment]
> Is someone currently working on this? If not I will gladly take this on, just let me know.

Yes, you can help me. 
But the organisation member/owners will decide, once they give us permission to officially work on this, then we'll be good to go.

**Comment by Dogeyyboi** [comment]
Ok so I have the link to tradingview.com right here

[https://www.tradingview.com/chart/Y4B4Puj1/](url)

**Comment by pouriya73** [comment]
> Ok so I have the link to tradingview.com right here
> 
> [https://www.tradingview.com/chart/Y4B4Puj1/](url)

One point must be considered
site tradingview
RAM uses a lot of system and it is better to use the following site coinmarketcap widget

**Comment by pouriya73** [comment]
Or API BINANCE . that written by PHP Language .

**Comment by abdoulayld** [comment]
@AkhileshThite    how can you  pay  the API fees?
